org.gustavoperedo.VideoDownloader.json

```diff
@@ -0,0 +1,116 @@
+{
+    "app-id" : "org.gustavoperedo.VideoDownloader",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "video-downloader",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.FileManager1",
+        "--filesystem=xdg-download/VideoDownloader:create",
+        "--metadata=X-DConf=migrate-path=/org/gustavoperedo/VideoDownloader/"
```

Comment:
I don't think it's needed. It's not as if you were using this name previously so there's nothing to migrate.